pom.xml

```diff
@@ -23,7 +23,7 @@
         <dependency>
             <groupId>io.appium</groupId>
             <artifactId>java-client</artifactId>
-            <version>4.1.2</version>
+            <version>5.0.0</version>
```

Comment:
Could we go straight to latest bugfix version `5.0.4`? https://mvnrepository.com/artifact/io.appium/java-client/5.0.4

Reply:
Maybe? I'll check. (For some reason I feel like that didn't exist yesturday?)